swarm-deploy.yml

```diff
@@ -206,7 +206,7 @@ services:
         - "traefik.http.routers.stt.tls=true"
         - "traefik.http.routers.stt.tls.certresolver=letsencrypt_resolver"
         - "traefik.http.services.stt.loadbalancer.server.port=8080"
-        - "traefik.http.routers.stt.priority=110"
+        - "traefik.http.routers.stt.priority=100"
```

Comment:
Useless to put different priorities for routers that are mutually exclusive, so just setting everything to 100 for clarity.